Frends.Template/setup.cmd

```diff
@@ -0,0 +1,10 @@
+@echo off
```

Comment:
This is windows specific, so it will not work on a Mac. I think that either this should be optional or then, if you cannot do optional, then just forget about this :)

Or maybe we could change into powershell or bash for xplat? Of course powershell is also not really guaranteed...